_data/where-we-work.yml

```diff
@@ -18,9 +16,5 @@
   position: [25%, 22%]
 - location: Woodbury, CT
   position: [25%, 21%]
-- location: Woodbury, CT
-  position: [25%, 21%]
-- location: Helsinki, Finland
-  position: [13%, 50.5%]
 - location: Berlin, Germany
   position: [20%, 48%]
```

Comment:
Ore, do you want to add:

- Calgary, Canada
- Chicago, Illinois
- San Francisco, CA
- Portland, OR

Also, I believe Amy works in Woodbury, CT.

Reply:
Ok, I'll add those. Woodbury, CT was duplicated so i just got rid of 1 entry.